keyboards/tapedeck/tapedeck.c

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2023 Gowla
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#ifdef POT_ENABLE
+  #include "analog.h"
+#endif
+
+int16_t pot_val = 0;
+int16_t prev_val = 0;
+int16_t val = 0;
+
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    if (!encoder_update_user(index, clockwise)) {
+        return false;
+    }
+    if (index == 0) { /* First encoder */
+        if (clockwise) {
+            tap_code(KC_WH_U);
+        } else {
+            tap_code(KC_WH_D);
+        }
+    } else if (index == 1) { /* Second encoder */
+        if (clockwise) {
+            tap_code(KC_BRIU);
+        } else {
+            tap_code(KC_BRID);
+        }
+    }
+    return true;
+}
+
+void matrix_init_kb(void) {
+#ifdef POT_ENABLE
+    analogReference(ADC_REF_POWER);
+#endif
```

Comment:
```suggestion
#endif
    matrix_init_user();
```

Reply:
Thank you for looking here :) I was told not to use _user function? Can this not be kept the same? https://github.com/qmk/qmk_firmware/pull/20161#discussion_r1140797395